cvmfs/options.h

```diff
@@ -15,6 +15,25 @@
 namespace CVMFS_NAMESPACE_GUARD {
 #endif
 
+/**
+ * Templating manager used for variable replacement in the config file
+ */
+class OptionsTemplatingManager{
+ public:
+  void SetVal(std::string name, std::string val);
+  std::string GetVal(std::string name);
+  bool HasVal(std::string name);
+ private:
+    std::map<std::string, std::string> vars;
+};
+
+class DefaultOptionsTemplatingManager : public OptionsTemplatingManager {
```

Comment:
So should the `OptionsTemplatingManager` then take the fqrn as a constructor argument or should I just use the "normal" `OptionsTemplatingManager` and then manually set the fqrn template variable?
I'm asking because I would have added the @org@ extraction logic to the `DefaultOptionsTemplatingManager` constructor otherwise.